cocos/renderer/gfx-gles2/GLES2PrimaryCommandBuffer.cpp

```diff
@@ -38,15 +38,13 @@
 namespace cc {
 namespace gfx {
 
-GLES2PrimaryCommandBuffer::GLES2PrimaryCommandBuffer()
-: GLES2CommandBuffer() {
-}
+GLES2PrimaryCommandBuffer::GLES2PrimaryCommandBuffer() = default;
```

Comment:
Why not define it in .h?

Reply:
https://clang.llvm.org/extra/clang-tidy/checks/modernize-use-equals-default.html